app.js

```diff
@@ -1,21 +1,38 @@
-var express=require('express');
-var expressLayouts = require('express-ejs-layouts');
-var session=require('express-session');
-var favicon=require('serve-favicon');
-var app=express();
-var trackController=require('./controllers/trackController');
-var loginController=require('./controllers/loginController');
-var bodyParser=require('body-parser');
-app.set('view engine','ejs');
-app.use(expressLayouts);
+const express=require('express');
+const expressLayouts = require('express-ejs-layouts');
+const app=express();
+const bodyParser=require('body-parser');
+
 app.use(express.static('./public'));
-app.use(express.json());
-app.use(bodyParser.json());
-app.use(bodyParser.urlencoded({extended:true}));
-trackController(app);
-loginController(app);
+app.use(bodyParser.urlencoded({extended:false}));
+
+// Passport
 const passport = require('passport');
 app.use(passport.initialize());
 app.use(passport.session());
-app.listen(4000);
-console.log('server listening');
\ No newline at end of file
+require('./config/passport')(passport);
+
+// EJS
+app.set('view engine','ejs');
+app.use(expressLayouts);
+
+// Routes
+const routes = require('./routes/routes');
+app.use(routes);
+
+/* MONGOOSE SETUP */
+const mongoose = require('mongoose');
+mongoose.connect('mongodb+srv://mihai:mihai@cluster0.j9czx.mongodb.net/musicapp?retryWrites=true&w=majority',{ 

```

Comment:
Picking up secrets from the environment is preferred. Committing to the repo should be avoided

Reply:
Thank you for mentioning that. Fixed it